cfme/tests/v2v/test_v2v_migrations_ui.py

```diff
@@ -241,3 +243,69 @@ def test_v2v_ui_set2(appliance, v2v_providers, form_data_single_datastore, soft_
     soft_assert(set(view.migration_request_details_list.read()) == set(vm_selected))
     view = navigate_to(infrastructure_mapping_collection, 'All')
     view.migration_plans_not_started_list.migrate_plan(plan_name)
+
+    # Test Archive completed migration  plan
+    wait_for(func=view.plan_in_progress, func_args=[plan_name], delay=5, num_sec=300,
+        handle_exception=True)
+    view.migr_dropdown.item_select("Completed Plans")
+    view.wait_displayed()
```

Comment:
I am suspect about such use of `wait_displayed` every time. I think `wt` internally take care of this.

Reply:
So these `view.wait_displayed()` are required here. When I navigate to this `view` initially, `wt` will do the wait by default as you said. Then what I do is, I am selecting different dropdowns in that view, and each dropdown item opens a different section, which shows `spinner` for brief duration, so using `view.wait_displayed()` I am really avoiding any operations before the spinner is gone. If I remove the wait_displayed() the test would pretty much be unreliable and fail inconsistently. You can try this on your local system and see what happens.